src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -312,6 +312,7 @@ void Graph::Replicate(const CNNNetwork &network) {
         for (size_t i = 0; i < childEdges.size(); i++) {
             const auto child = childEdges[i]->getChild();
             if (child->getOriginalInputPrecisionAtPort(childEdges[i]->getOutputNum()) != Precision::BF16 &&
+                child->getOriginalInputPrecisionAtPort(childEdges[i]->getOutputNum()) != Precision::FP16 &&
```

Comment:
Please use a variable for
child->getOriginalInputPrecisionAtPort(childEdges[i]->getOutputNum()

Reply:
fixed